nilearn/decoding/tests/test_space_net.py

```diff
@@ -38,7 +38,7 @@
 
 
 @pytest.mark.parametrize("is_classif", IS_CLASSIF)
-@pytest.mark.parametrize("l1_ratio", [0.5, 1.0])
+@pytest.mark.parametrize("l1_ratio", [0.5, 0.99])
```

Comment:
using a value of 0 or 1 triggers a warning

Reply:
Yes, but... isn't it because 1.0 at the limit of the parameter domain ?
It could be that using 1.0 triggers another optimization approach (if not, forget my comment).
We need to check whether having the parameter equal to 0 or 1 triggers a special behavior in the ensuing optimization.

Reply:
Sorry if I was unlclear, there are still some tests that use 0 or 1 that I have not touched that are actual proper check of the behavior for those limit values:

- test_lasso_vs_graph_net
- test_log_reg_vs_graph_net_two_classes_iris
- test_space_net_alpha_grid_pure_spatial

I only changed the values that were used in the test parametrizations.